packages/fx-core/src/plugins/resource/appstudio/plugin.ts

```diff
@@ -1697,8 +1678,33 @@ export class AppStudioPluginImpl {
       }
     }
 
-    for (const domain of validDomains) {
-      updatedManifest.validDomains?.push(domain);
+    // This should be removed in future, the valid domains will be rendered by states
+    if (updatedManifest.validDomains?.length == 0) {
```

Comment:
Will this change affect existing projects since there's no placeholder for them?